src/default_job_handlers/recurring_job/__init__.py

```diff
@@ -1,9 +1,11 @@
+import datetime
 from typing import Tuple
 
 from services.dmss import add_document
 from services.job_handler_interface import Job, JobHandlerInterface, JobStatus
-from services.job_service import register_job
+from services.job_service import register_job, remove_job
 from utils.logging import logger
+from utils.timezone import getTimezoneAwareCurrentDatetime
```

Comment:
Don't think you need this function, nor the pytz package.
All timestamps in DMSS are stored as UTC+0/Zulu, always. So no timezone info needed